pages/templates/partials/past_results.html

```diff
@@ -36,31 +36,25 @@ <h2 class="mb20">
           </a>
           {% endif %}
 
-          <hr>
+          {% comment %}Wagtail places HTML tags in blank RichTextFields, so a
```

Comment:
Thanks for added the code comments to explain this. Another option is to add a boolean checkbox to toggle display of content.